system/modules/isotope/library/Isotope/Model/Gallery/Standard.php

```diff
@@ -357,7 +357,18 @@ protected function getImageForType($strType, array $arrFile, $blnWatermark = tru
 
         try {
             $strImage = \Image::create($strFile, $size)->executeResize()->getResizedPath();
-            $picture = \Picture::create($strFile, $size)->getTemplateData();
+
+            if (!is_file(TL_ROOT . '/' . $strImage)) {
+                $image = \System::getContainer()->get('contao.image.image_factory')->create(TL_ROOT . '/' . $strImage);
+                \System::getContainer()->get('contao.image.resizer')->resizeDeferredImage($image);
+            }
+
+            $picture = \Picture::create($strFile, $size)->getTemplateData();            
+
+            if (!is_file(TL_ROOT . '/' . $picture['img']['src'])) {
+                $src = \System::getContainer()->get('contao.image.image_factory')->create(TL_ROOT . '/' . $picture['img']['src']);
+                \System::getContainer()->get('contao.image.resizer')->resizeDeferredImage($src);
+            }
```

Comment:
What about images in a `srcset` or multiple `<source>`s? I mean, not that it really matters for PDF creation, but technically all the sources would neet to be resized in advance.

/cc @ausi

Reply:
`srcset` is used only in browsers. There we don't have any problems. The problem is only in PDF generation.